src/hooks/useHandleCreateWrapped.tsx

```diff
@@ -82,6 +86,76 @@ import { useTerraWallet } from "../contexts/TerraWalletContext";
 import { TerraWallet } from "@xlabs-libs/wallet-aggregator-terra";
 import { XplaWallet } from "@xlabs-libs/wallet-aggregator-xpla";
 import { useXplaWallet } from "../contexts/XplaWalletContext";
+import {
+  JsonRpcProvider,
+  SUI_CLOCK_OBJECT_ID,
+  TransactionBlock,
+  getPublishedObjectChanges,
+} from "@mysten/sui.js";
+import {
+  getPackageId,
+  getWrappedCoinType,
+} from "@certusone/wormhole-sdk/lib/cjs/sui";
+import { getSuiProvider } from "../utils/sui";
+import { sleep } from "../utils/sleep";
+import { useSuiWallet } from "../contexts/SuiWalletContext";
+import { SuiWallet } from "@xlabs-libs/wallet-aggregator-sui";
+import { createWrappedOnSuiPrepare } from "../utils/suiPublishHotfix";
+
+// TODO: replace with SDK method
+export async function updateWrappedOnSui(
```

Comment:
Is this method already in the sdk (see [here](https://github.com/wormhole-foundation/wormhole/blob/sui/integration_v2/sdk/js/src/token_bridge/updateWrapped.ts#L45))? I don't know which version the bet package was built from though, so this might not be the case

Reply:
Unfortunately not released yet 😢